docs/specs/redirection.yml

```diff
@@ -564,3 +564,53 @@ inputs:
       ]
     }
 outputs:
+---
+# Support source_monikers for rediction
+repos:
+  https://github.com/redirection/source-monikers:
+  - files:
+      docfx.yml: |
+        monikerDefinition: monikerDefinition.json
+      .openpublishing.redirection.json: |
+        { "redirections": [{"source_path": "a.md", "monikers": ["netcore-1.0", "netcore-2.0"], "redirect_URL": "/b"}, {"source_path": "a.md", "monikers": "netcore-2.1", "redirect_URL": "/c"}] }
```

Comment:
nit: format it a little bit to show there are two lines of redirections

consider also adding a negative case of two redirections of the same path with overlapping monikers